src/python/pants/util/desktop.py

```diff
@@ -35,7 +37,7 @@ def _cmd_exists(cmd):
 }
 
 
-def ui_open(*files):
+def ui_open(*files: str) -> None:
```

Comment:
Huh, this is a strange way of representing this. I understand that the type annotation means "zero or more arguments, all of which are strings", but in the code the name `files` will be bound to a `List[str]`, not a `str`.

Not saying this is wrong, just a small corner of python typing I find odd :)

Reply:
I think technically `files` is a `Sequence[str]`, which is the metaclass(?) for `List`. 

Agreed, though, that is strange. Kwargs is the same:

```
def my_function(**kwargs: Union[str, int]):
 ...
```